app/src/main/java/com/mapbox/maps/testapp/examples/terrain3D/SantaCatalinaActivity.kt

```diff
@@ -41,8 +41,8 @@ class SantaCatalinaActivity : AppCompatActivity() {
   private var timeAnimator: TimeAnimator? = null
 
   override fun onCreate(savedInstanceState: Bundle?) {
-    super.onCreate(savedInstanceState)
     requestWindowFeature(Window.FEATURE_NO_TITLE)
+    super.onCreate(savedInstanceState)
```

Comment:
could you explain this change?

Reply:
robo-test found crash if the order leave without changes

Reply:
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.mapbox.maps.testapp/com.mapbox.maps.testapp.examples.terrain3D.SantaCatalinaActivity}: android.util.AndroidRuntimeException: requestFeature() must be called before adding content
```